src/gg20/sign/r8/happy.rs

```diff
@@ -82,25 +82,19 @@ impl Executer for R8Happy {
         // compute s = sum_i s_i
         let s = bcasts_in
             .iter()
-            .fold(Scalar::zero(), |acc, (_, bcast)| acc + bcast.s_i.as_ref());
+            .fold(Scalar::ZERO, |acc, (_, bcast)| acc + bcast.s_i.as_ref());
 
         let sig = {
-            let mut sig = Signature::from_scalars(self.r, s).map_err(|_| {
+            let sig = Signature::from_scalars(self.r, s).map_err(|_| {
                 error!("scalars to signature conversion failed");
                 TofnFatal
             })?;
-
```

Comment:
This could be insecure, but `normalize_s` changed, so I modified this check, removing the fatal error